src/scripts/clipperUI/components/sectionPicker.tsx

```diff
@@ -288,10 +288,9 @@ export class SectionPickerClass extends ComponentBase<SectionPickerState, Sectio
 			}
 		}
 
-		// Display an un-retryable message if API responds with a code that is not retryable
-		// Note that if this is an unhandled api response code, we assume it is not retryable
-		if (this.state.apiResponseCode && !OneNoteApiUtils.isRetryable(this.state.apiResponseCode)) {
-			localizedStrings.notebookLoadFailureMessage = Localization.getLocalizedString("WebClipper.SectionPicker.NotebookLoadUnretryableFailureMessage");
+		// If we can show a better message, especially an actionable one, we do
+		if (this.state.apiResponseCode && OneNoteApiUtils.getLocalizedErrorMessage(this.state.apiResponseCode)) {
```

Comment:
I removed it because I figured that with the sectionPicker, there's no concept of 'retrying' a GetNotebooks call. Would it be a better experience if we show the retryable message for retryable codes anyway?